private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -461,45 +807,188 @@ export class TemplateApi {
         'bearer_auth': new ApiKeyAuth('header', 'Authorization'),
     }
 
-    constructor(basePath?: string);
-    constructor(basePathOrUsername: string, password?: string, basePath?: string) {
-        if (password) {
-            if (basePath) {
-                this.basePath = basePath;
-            }
-        } else {
-            if (basePathOrUsername) {
-                this.basePath = basePathOrUsername
-            }
+    constructor(basePath?: string);
+    constructor(basePathOrUsername: string, password?: string, basePath?: string) {
+        if (password) {
```

Comment:
if(password && basepath)

Reply:
wait is this file generate?

Reply:
Yep, this file was generated from the yaml definition!